client/app/assets/less/ant.less

```diff
@@ -391,6 +391,18 @@
         left: 0;
```

Comment:
That's complicated. I did think about it, but the thing is that this PR resulted in a lot of visual changes, which I had to solve in order to be mergeable. What is sort of possible but somewhat weird is having the fixes for those in master before we merge it. LMK what is your opinion on that.

An alternative is to review those last commits primarily, since they are where the styling changes are amassed.

Reply:
What if we just did one button type at a time?